### ISSUE TITLE
Does not check words longer than 39 chars

### ISSUE THREAD

**Issue by alternatiph** [streetsidesoftware/vscode-spell-checker#693]
I think the screenshot pretty much sums up the issue.

```
thisIsAStringToCheckSpellChekAddon: string
thisIsAStringToCheckSpellChekAddonFunctionality: string
```
<img width="622" alt="Screen Shot 2021-02-02 at 3 45 19 PM" src="https://user-images.githubusercontent.com/11921744/106598939-a8a29500-656d-11eb-9abe-0dda1e985337.png">

**Comment by Jason3S** [comment]
You are right. It is a current limitation of the spell checker to avoid false positives. One option would be to make it configurable.

**Comment by alternatiph** [comment]
@Jason3S If you could give me a hint of where to look at the codes I would be happy to create a PR for this and make it configurable.

**Comment by Jason3S** [comment]
It is the  `Base64` detection that is causing it.

A simple workaround is to define a new pattern in a `cspell.json` file:

```
    "patterns": [
        {
            "name": "Base64",
            "pattern": "/$^/g"
        }
    ],
```

or VS Code Settings:

```
"cSpell.patterns": [
	{
		"name": "Base64",
		"pattern": "/$^/g"
	}
]
```

This will effectively turn off Base64 detection.

**Comment by Jason3S** [comment]
I would welcome a PR if you would like to come up with a fast RegExp that matches Base64 without matching long variable or function names.

The current RegExp is here: [cspell/RegExpPatterns.ts at master · streetsidesoftware/cspell](https://github.com/streetsidesoftware/cspell/blob/master/packages/cspell-lib/src/Settings/RegExpPatterns.ts#L14)
